src/seedu/addressbook/data/person/address/Address.java

```diff
@@ -9,13 +9,25 @@
 public class Address {
 
     public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must be in the format"
+            + " BLOCK, STREET, UNIT, POSTAL_CODE";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
 
+    private final static String EMPTY_STRING = "";
+    private final static int NUMBER_OF_ATTRIBUTES = 4;
+
     public final String value;
```

Comment:
What is the purpose of maintaining this value now that you have small components?